content/docs/pipelines/tutorials/cloud-tutorials.md

```diff
@@ -9,4 +9,5 @@ weight = 1
 See the <a href="/docs/about/docs/">guide to the Kubeflow docs</a>.</p>
 {{% /alert %}}
 
-* [Pipelines End-to-end on Azure](/docs/azure/azureendtoend/): An end-to-end tutorial for Kubeflow Pipelines on Microsoft Azure.
\ No newline at end of file
+* [Pipelines End-to-end on Azure](/docs/azure/azureendtoend/): An end-to-end tutorial for Kubeflow Pipelines on Microsoft Azure.
+* [Pipelines - GitHub Issue Summarization](https://g.co/codelabs/kfp-gis) : An end-to-end tutorial for Kubeflow deployment both from  Pipelines UI, and from a Jupyter Notebook.
```

Comment:
Thanks for doing this. How about a slightly modified blurb that indicates it's a GCP example, and gives a bit more detail?
"This GCP tutorial walks through a Kubeflow Pipelines example that shows training a Tensor2Tensor model for GitHub issue summarization, both via the Pipelines Dashboard UI, and from a Jupyter notebook."

Reply:
Yes of course, I will add your comments. I just thought more detail was not needed judging from the tutorial description on Azure which was small.